### PR TITLE
mds: increase default cache memory limit to 4GB

### PR DESCRIPTION
**Pull request by batrick**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by jtlayton** [comment]
@batrick : I think this needs a tracker ticker, no?

**Comment by batrick** [comment]
> @batrick : I think this needs a tracker ticker, no?

Yes, thanks! Updated.

**Comment by batrick** [comment]
> One question though: is it documented somewhere that the host running the MDS ought to dedicate at least this much memory to its use?

I've updated the docs.

File: doc/cephfs/add-remove-mds.rst [hunk 1]
```diff
@@ -15,9 +15,9 @@ Provisioning Hardware for an MDS
 ================================
 
 The present version of the MDS is single-threaded and CPU-bound for most
-activities, including responding to client requests. Even so, an MDS under the
-most aggressive client loads still uses about 2 to 3 CPU cores. This is due to
-the other miscellaneous upkeep threads working in tandem.
+activities, including responding to client requests. An MDS under the most
+aggressive client loads uses about 2 to 3 CPU cores. This is due to the other
+miscellaneous upkeep threads working in tandem.
```
Comment (jtlayton) [inline doc/cephfs/add-remove-mds.rst]: Note that these comments will likely bitrot over time. CPU cores today are much faster than the ones from just a few years ago.

**Pull request merged**